feat: add manual sync functionality and localization updates for PrimeCloudSync OK-43224

yikZero opened the pull request:
- Introduced a manual sync feature in the PrimeCloudSync page, allowing users to trigger cloud synchronization directly.
- Added a new translation key for "Back up now" to enhance user prompts related to wallet backup.
- Updated localization files to include the new sync success message in multiple languages, improving user experience across different locales.
- Enhanced existing translations for better clarity and consistency in the Prime feature set.

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * Added a manual Cloud Sync trigger in App Data settings (shown when Cloud Sync is enabled). Prevents concurrent manual syncs, shows a loading dialog during sync, updates last sync time, and displays a success confirmation.

* **Localization**
  * Added translation keys for: sync success, wallet “Back up now,” identical-name asset alert, and perpetuals (trade margin type, trade order type).
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

yikZero committed:
feat: add manual sync functionality and localization updates for PrimeCloudSync

- Introduced a manual sync feature in the PrimeCloudSync page, allowing users to trigger cloud synchronization directly.
- Added a new translation key for "Back up now" to enhance user prompts related to wallet backup.
- Updated localization files to include the new sync success message in multiple languages, improving user experience across different locales.
- Enhanced existing translations for better clarity and consistency in the Prime feature set.

revan-zhang commented:
### :tada: **Snyk checks have passed. No issues have been found so far.**

:white_check_mark: **security/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/rayston92/pr-checks/32425abb-5080-47cc-8741-190bd6abe504?refs=7637ad0242d48b862b36d4a17aaff3f183f38a1c&source=prChecksComment)

:white_check_mark: **license/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/rayston92/pr-checks/32425abb-5080-47cc-8741-190bd6abe504/license?refs=7637ad0242d48b862b36d4a17aaff3f183f38a1c&source=prChecksComment)

yikZero committed:
Merge branch 'x' into prime-backup-now

limichange commented on packages/kit/src/views/Prime/pages/PrimeCloudSync/PagePrimeCloudSync.tsx:
```diff
@@ -325,10 +326,53 @@ function AppDataSection() {
   }, [config.lastSyncTime]);
 
+  const handleManualSync = useCallback(async () => {
+    if (!config.isCloudSyncEnabled) {
```
没开启的话，点击给一个提示？

sidmorizon replied:
没事，没开启的话，这个按钮看不到

yikZero replied:
> 没开启的话，点击给一个提示？

这个按钮只在开关打开的时候展示，没开启用户看不到按钮没法点。

`{config?.isCloudSyncEnabled ? (
        <ListItem
          title={intl.formatMessage({
            id: ETranslations.wallet_backup_now,
          })}
          icon="RefreshCwOutline"
          drillIn
          onPress={handleManualSync}
        />
      ) : null}`

yikZero committed:
refactor: remove setUndefinedTimeToNow parameter from manual cloud sync flow

- Eliminated the setUndefinedTimeToNow parameter from the startServerSyncFlow method in the PrimeCloudSync page, streamlining the cloud synchronization process.
- This change enhances code clarity and maintains the existing functionality of the manual sync feature.

yikZero committed:
Merge branch 'x' into prime-backup-now

yikZero committed:
feat: adjust localization

originalix commented:
处理一下冲突

yikZero committed:
Merge branch 'x' into prime-backup-now

limichange committed:
Merge branch 'x' into prime-backup-now

yikZero committed:
Merge branch 'x' into prime-backup-now

yikZero committed:
Merge branch 'x' into prime-backup-now

sidmorizon committed:
Merge branch 'x' into prime-backup-now
